src/plugins/MoreSearchOperators.ts

```diff
@@ -0,0 +1,213 @@
+            let value = match.getMatch(1);
+            switch (value) {
+                case "relevance":
+                    match.setData("sortBy", "relevance");
+                    return true;
+                case "timestamp":
+                    match.setData("sortBy", "timestamp");
+                    return true;
+                default:
+                    return false;
+            }
+        },
+        follows: ["FILTER_SORT_BY"],
+        queryKey: "sort_by",
+        mutable: true,
+        _dataKey: "sortBy"
+    },
+    ANSWER_FILE_NAME: {
+        componentType: "ANSWER",
+        regex: /(?:\s*([^\s]+))/,
+        validator: function (match) {
+            match.setData("fileName", match.getMatch(1));
+            return true;
+        },
+        follows: ["FILTER_FILE_NAME"],
+        queryKey: "attachment_filename",
+        mutable: true,
+        _dataKey: "fileName"
+    },
+    ANSWER_FILE_TYPE: {
+        componentType: "ANSWER",
+        regex: /(?:\s*([^\s]+))/,
+        validator: function (match) {
+            match.setData("fileType", match.getMatch(1));
+            return true;
+        },
+        follows: ["FILTER_FILE_TYPE"],
+        queryKey: "attachment_extension",
+        mutable: true,
+        _dataKey: "fileType"
+    },
+    ANSWER_EMBED_TYPE: {
+        componentType: "ANSWER",
+        regex: /\s*([^\s]+)/i,
+        validator: function (match) {
+            match.setData("embedType", match.getMatch(1));
+            return true;
+        },
+        follows: ["FILTER_EMBED_TYPE"],
+        queryKey: "embed_type",
+        mutable: true,
+        _dataKey: "embedType"
+    },
+};
+
+export default definePlugin({
+    name: "MoreSearchOperators",
+    description: "Adds experimental search operators.",
+    authors: [Devs.Davri],
+
+    patches: [{
+        find: "Messages.SEARCH_SHORTCUT",
+        replacement: {
+            match: /(.)\((.),.\..{1,3}\.ANSWER_PINNED/,
+            replace: Object.keys(searchOperators).map(a => `$1($2,"${a}",$self.searchOperators.${a}),`).join("") + Object.keys(searchAnswers).map(a => `$1($2,"${a}",$self.searchAnswers.${a}),`).join("") + "$&"
+        }
+    }, {
+        find: "Messages.SEARCH_ANSWER_FROM",
+        replacement: {
+            match: /var .=.\[.\];switch\(.\)\{/,
+            replace: "$&" + Object.entries(searchAnswers).map(([k, v]) => `case "${k}":a.add(e.getData("${v['_dataKey']}"));break;`).join("")
+        }
+    },
+    {
+        find: "Messages.SEARCH_ANSWER_FROM",
+        replacement: {
+            match: /function .\(.\)\{switch\(.\)\{/,
+            replace: "$&" + Object.entries(searchOperators).map(([k, v]) => `case \"${k}\":return \"${v['_options']}\";`).join("")
+        }
+    },
+    {
+        find: "Messages.SEARCH_GROUP_HEADER",
+        replacement: {
+            match: /(.{2})\((.{2}),.\..{3}\.FILTER_HAS,\{titleText:function\(\)\{return .\..\.Messages.SEARCH_GROUP_HEADER_HAS\}\}\)/,
```

Comment:
```suggestion
            match: /(\i)\((\i),\i\.\i\.FILTER_HAS,\{titleText:function\(\)\{return \i\.\i\.Messages.SEARCH_GROUP_HEADER_HAS\}\}\)/,
```
@Vendicated we need brace balancing so baddd